plugins/lua/spectate.lua

```diff
@@ -21,32 +27,69 @@ local function get_default_state()
         ['include-wildlife']=false,
         ['prefer-conflict']=true,
         ['prefer-new-arrivals']=true,
+        ['tooltip-follow-blink-milliseconds']=2000,
```

Comment:
all new user-settable settings will need docs in `docs/plugins/spectate.rst`

Reply:
done